sdcm/cluster_k8s/mini_k8s.py

```diff
@@ -484,10 +484,14 @@ def node_terminate_methods(self) -> List[str]:
             return ['drain_k8s_node']
         return []
 
-    @retrying(n=20, sleep_time=60, allowed_exceptions=(cluster.ClusterNodesNotReady, UnexpectedExit),
-              message="Waiting for nodes to join the cluster")
-    def wait_for_nodes_up_and_normal(self, nodes=None, verification_node=None):
-        super().wait_for_nodes_up_and_normal(nodes, verification_node)
+    def wait_for_nodes_up_and_normal(self, nodes=None, verification_node=None, iterations=60, sleep_time=3,  timeout=0):  # pylint: disable=too-many-arguments
```

Comment:
It was 20 minutes before the change and now it is only 3 minutes.
Considering that local K8S may run on some slow node I expect it to be failed in lots of cases.
Please, keep 20 minutes as it was before.

Reply:
Sorry, my fault, thanks. fixed